lib/Basics/process-utils.cpp

```diff
@@ -97,6 +96,21 @@ using namespace arangodb;
 
 namespace {
 
+#ifdef _WIN32
+HANDLE getProcessHandle(TRI_pid_t pid) {
```

Comment:
This function code almost same for TRI_LookupSpawnedProcessStatus, can you reuse it?

Reply:
no. The reason to split this was to return simple data types, so the locking is working.
This is just the windows process handle.

Reply:
Ok